distribution/bwc/build.gradle

```diff
@@ -169,8 +169,9 @@ bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInfo unreleased
                         'JAVA_HOME',
                         getJavaHome(it, Integer.parseInt(
                                 lines
-                                        .findAll({ it.startsWith("ES_BUILD_JAVA=java") })
+                                        .findAll({ it.startsWith("ES_BUILD_JAVA=")})
```

Comment:
should the ES_RUNTIME_JAVA be updated too?
Should we also update the .ci/java-versions.properties ?

Reply:
also should we change this line below?
`                                        .collect({ it.replace("ES_BUILD_JAVA=java", "").trim() })
`